api/db-cdr/generate-cdr/create-tables.sh

```diff
@@ -1,77 +1,69 @@
-#!/usr/bin/env bash
-
-# using #!/usr/bin/env bash else need to change for local development
-# #!/usr/bin/env bash -> we can use associative arrays
-# #!/bin/bash -> we can use associative arrays
+#!/bin/bash
 
 # This script removes/creates all CDR indices specific tables.
+
 set -e
 
 export BQ_PROJECT=$1         # CDR project
 export BQ_DATASET=$2         # CDR dataset
-export CREATE_SURVEYS=$3     # Create surveys flag
 
-function deleteAndCreateTable(){
-  local table_name=$1
-  # delete table - no error thrown if table does not exist
-  echo "Deleting $table_name"
-  bq --project_id="$BQ_PROJECT" rm -f "$BQ_DATASET.$table_name"
-  wait
-  # if number of arguments = 2, create a clustered table
-  if [[ $# -eq 2 ]]; then
-    local cluster_fields=$2
-    echo "Creating $table_name clustering_fields: $cluster_fields"
-    bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" --time_partitioning_type=DAY --clustering_fields "$cluster_fields" "$BQ_DATASET.$table_name"
-  else
-    # create a basic table
-    echo "Creating $table_name"
-    bq --quiet --project_id="$BQ_PROJECT" mk --schema="$schema_path/$json_name" "$BQ_DATASET.$table_name"
-  fi
-  wait
-}
+TABLE_LIST=$(bq ls -n 1000 "$BQ_PROJECT:$BQ_DATASET")
 
-INCOMPATIBLE_DATASETS=("R2019Q4R3" "R2019Q4R4" "R2020Q4R3", "R2021Q3R5", "C2021Q2R1", "C2021Q3R6", "R2022Q2R2", "C2022Q2R2", "R2022Q2R6", "SR2022Q2R6", "SC2022Q2R6", "SC2021Q2R1")
+INCOMPATIBLE_DATASETS=("R2019Q4R3" "R2019Q4R4" "R2020Q4R3")
 
 if [[ ${INCOMPATIBLE_DATASETS[@]} =~ $BQ_DATASET ]];
   then
   echo "Can't run CDR build indices against $BQ_DATASET!"
   exit 1
 fi
-
-TABLE_LIST=$(bq ls -n 1000 "$BQ_PROJECT:$BQ_DATASET" | tail -n +3 | cut -d " " -f 3 )
-
-SKIP_TABLES=("cb_data_filter" "cb_person" "survey_module" "domain_card")
-
-declare -A CLUSTERED_TABLES
-CLUSTERED_TABLES["cb_search_all_events"]="concept_id"
-CLUSTERED_TABLES["cb_review_survey"]="person_id"
-CLUSTERED_TABLES["cb_search_person"]="person_id"
-CLUSTERED_TABLES["cb_review_all_events"]="person_id,domain"
-
+done_ds_proc_occur=0
 schema_path=generate-cdr/bq-schemas
 for filename in generate-cdr/bq-schemas/*.json;
 do
     json_name=${filename##*/}
     table_name=${json_name%.json}
+    if [[ "$table_name" != 'ds_procedure_occurrence_52' ]]
+    then
+      echo "Deleting $table_name"
+      bq --project_id="$BQ_PROJECT" rm -f "$BQ_DATASET.$table_name"
+    fi
```

Comment:
Need to double check if bq-schemas contain the ds_procedure_occurrence_52.json, as it is not present in main

Reply:
Thanks for noticing this. Added.